apps/site/src/app/schedule/DayThree/DayThree.module.scss

```diff
@@ -0,0 +1,77 @@
+@use "bootstrap-utils" as utils;
+
+.mapContainer {
+	width: 100vw;
+	position: relative;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	height: auto;
+	height: 1300px;
+}
+
+.map3 {
+	position: absolute;
+	width: 120%;
+	min-width: 1400px;
+	height: auto;
+}
+
+.title {
+	position: absolute;
+	top: -250px;
+	left: 50%;
+	transform: translateX(-50%);
+}
+
+.mapPath {
+	position: absolute;
+}
+
+.pathImage {
+	position: relative;
+	top: -160px;
+	width: 70%;
+	height: 160%;
+	left: 15%;
+}
+
+@include utils.media-breakpoint-up(sm) {
+	.pathImage {
+		top: 0;
+		left: 0;
+		width: 100%;
+		height: 100%;
+	}
+
+	.map3 {
+		position: absolute;
+		width: 80%;
+		min-width: 1100px;
+	}
+
+	.title {
+		top: -100px;
+	}
+}
+
+@media screen and (min-width: 1751px) {
```

Comment:
suggestion (repeating): these media queries could be replaced with the bootstrap media breakpoints. For example [here](https://github.com/HackAtUCI/zothacks-site/blob/4366ce4c9124b948ace5e79b499a1fa9fc2319f4/apps/site/src/app/(home)/sections/GetInvolved/GetInvolved.module.scss#L19).